Sources/ComposableArchitecture/SwiftUI/ActionSheet.swift

```diff
@@ -160,19 +160,6 @@ extension ActionSheetState: Equatable where Action: Equatable {
   }
 }
 
-@available(iOS 13, *)
-@available(macCatalyst 13, *)
-@available(macOS, unavailable)
-@available(tvOS 13, *)
-@available(watchOS 6, *)
-extension ActionSheetState: Hashable where Action: Hashable {
```

Comment:
`Animation` is not hashable, so we lose this conformance...unless you have ideas?

Reply:
I think we could just leave `Animation` out of the hashing. it means a slightly less performant hash but still semantically correct.